src/server/ApiRoutes/Search.js

```diff
@@ -81,7 +81,13 @@ function searchServerPost(req, res) {
   }
 
   if (!searchKeywords) {
-    return res.redirect(`${appConfig.baseUrl}/search?error=true`);
+    return res.redirect(`${appConfig.baseUrl}/search?error=noKeyword`);
+  }
+
+  if (dateAfter && dateBefore) {
+    if (Number(dateAfter) > Number(dateBefore)) {
```

Comment:
I think you mean if the date before is > than the date after, then redirect to the error.

Reply:
No. dateAfter means the start date and dateBefore means the end date, just as the filters name. Just as the check in FilterPopup

Reply:
True. 👍